### ISSUE TITLE
cannot control Lite3 to move

### ISSUE THREAD

**Issue by Zvch17** [DeepRoboticsLab/Lite3_Model_Control#2]
Hi, 

I'm trying to use high_level_sim to simulate Lite3's movements in Gaezbo. I followed the Simulation_Manual.md in high_level_sim, successfully starting the Gazebo simulator and loading a robot, but I can not control the robot to move as I run example_keyboard.
How can I correctly control the robot with the keyboard?

**Comment by yyang0802** [comment]
Please check if the **Enter** key is pressed before running example_keyboard.


![image](https://github.com/user-attachments/assets/89de5ee5-f2a5-41b1-8395-e0f97cc19213)

**Comment by Zvch17** [comment]
Thanks for reply. I have checked that I pressed the Enter key. Here is what I have done:
1. run roslaunch gazebo_model_spawn gazebo_startup.launch in a new terminal and I got an empty world in gazebo
2. run roslaunch gazebo_model_spawn model_spawn.launch in a new terminal and I got a Lite3 robot in gazbeo
3. press Enter key in the Step 2's terminal and I got the return showing every contrller correctly loaded
![image](https://github.com/user-attachments/assets/b84080ad-d711-4f74-a0a9-707c019a2273)
4. run the rosrun examples example_lite3_sim in a new terminal, the robot stood up and the terminal returned data as following:
![image](https://github.com/user-attachments/assets/79148207-1bac-4910-9943-2e86a8dacd96)
5. run rosrun examples example_keyboard in a new terminal, when I pressed w, a, s, d, the data on the '/joy' rostopic changed, but the robot didn't move. 

Also, I tried to stop the example_lite3_sim and use example_keyboard only, and the data on the '/joy' rostopic changed while the robot didn't move. I tried to check the code of example_lite3_sim but it is complicated. Is it because the data on 'joy' needs to be translated to other types of data, but example_keyboard acutually does not do this?
